arlo.py

```diff
@@ -828,15 +828,25 @@ def Geofencing(self, location_id, active=True):
         """
         return self.request.put('https://my.arlo.com/hmsweb/users/locations/'+location_id, {'geoEnabled':active})
 
-    def GetDevices(self, device_type=None):
+    def GetDevices(self, device_type=None, filter_provisioned=None):
         """
         This method returns an array that contains the basestation, cameras, etc. and their metadata.
-        If you pass in a valid device type ('basestation', 'camera', etc.), this method will return an array of just those devices that match that type.
+        If you pass in a valid device type, as a string or a list, this method will return an array of just those devices that match that type. An example would be ['basestation', 'camera']
+        To filter provisioned or unprovisioned devices pass in a True/False value for filter_provisioned. By default both types are returned. 
         """
         devices = self.request.get('https://my.arlo.com/hmsweb/users/devices')
         if device_type:
-            return [ device for device in devices if device['deviceType'] == device_type]
+            if not isinstance(device_type,list):
```

Comment:
Can you please add a space after that `,`. Thanks!

Reply:
Actually, you don't need to do that `if not isinstance` at all. Python will do the right thing whether `device_type` is a string or array. Get rid of lines 839 and 840 and test it out for yourself.

Reply:
Nice. I didn't know that. I figured that it would treat the string as an array in that case, like ```['c','a','m','e','r','a']```. Nice to know.

Reply:
Python is 🆒. Thanks for the contribution!